src/index.js

```diff
@@ -92,11 +90,36 @@ module.exports = function validate(config, options = {}) {
     process.exit(1)
   }
 
+  return config
+}
+
+// Easier consumability for require (default use case for non-transpiled webpack configs)
+function validateRoot(config, options = {}) {
+  const {
+    quiet,
+  } = options
+
+  let validationResult,
+    multiValidationResults
+
+  if (Array.isArray(config)) {
+    multiValidationResults = []
+    config.forEach((cfg) => {
+      multiValidationResults.push(
+        validate(cfg, options)
+      )
+    })
+  } else {
+    validationResult = validate(config, options)
+  }
+
   if (!quiet) {
     console.info(chalk.green('[webpack-validator] Config is valid.'))
   }
 
-  return config
+  return validationResult || multiValidationResults
 }
 
-module.exports.Joi = Joi
```

Comment:
Would like to take a stab at this since it was my code that broke it, but unfortunately I will be away from a computer for the next 8 hours ago. LMK how urgent this is. I could submit a PR tomorrow if that's not too late.